source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
-                     <head>Cutouts</head>
+                  <div xml:id="sourcesPrinted" type="div4">
+                     <head>Additional for describing printed sources</head>
+                     <p></p>
                      <p>
                         <specList>
-                           <specDesc key="cutout"/>
+                           <specDesc key="plateNum"/>
+                           <specDesc key="typeDesc"/>
                         </specList>
                      </p>
-                     <p>Cutouts are treated almost similarly as <ptr target="#patches"/>. The most relevant attributes are:</p>
+                     <p>For <gi scheme="MEI">captureMode</gi> see <ptr target="#audioAndOther"/>.</p>
+                  </div>
+                  <div xml:id="addDesc" type="div4">
+                     <head>Additional Material/Information</head>
+                     <p></p>
                      <p>
                         <specList>
-                           <specDesc key="cutout" atts="removed.from"/>
-                           <specDesc key="cutout" atts="removed.by"/>
+                           <specDesc key="accMat"/>
+                           <specDesc key="addDesc"/>
+                           <specDesc key="catchwords"/>
                         </specList>
                      </p>
-                     <p>The dimensions (<att>width</att>, <att>height</att>) of the parent element (<abbr>e.g.</abbr>, <gi scheme="MEI">folium</gi>) indicate the size of the bounding box of the remaining part of the page. That is, if the complete lower half of a page has been cut, the <att>width</att> and <att>height</att> attributes describe the remaining upper half. If, in contrast, only the lower right quarter of the page has been cut, these attributes still indicate the size of the full page (assuming that the removed section was a regular rectangle).The dimensions (<att>width</att>, <att>heigh</att>) on cutout itself are only to be used when there is a ‘gap’ in the manuscript that allows to specify the dimensions of that missing part. In this case, the bounding box dimensions are given, together with <att>x</att> and <att>y</att> to indicate the upper left point on the original page. If, however, the removed section is available by itself, then a corresponding <gi scheme="MEI">folium</gi> (or <gi scheme="MEI">bifolium</gi>) should be placed inside the cutout element, and should provide it’s own dimensions using <att>width</att> and <att>height</att> there. In this case, <att>width</att> and <att>height</att> on cutout is expendable.</p>
-                     <p>The genetic aspect of applying patches or cutting out parts of a page is described in <ptr target="#genetic"/>.</p>
+                     <p></p>
                   </div>
+                  <div xml:id="medievalManuscripts" type="div4">
+                     <head>Additional for describing Medieval manuscripts</head>
```

Comment:
I would remove "Medieval" here, since there is nothing that is exclusively medieval -- heraldry, seals, rubrics, decoration, etc. can be found in manuscripts well beyond the medieval period.

Reply:
+1 for removing `medieval`

Reply:
also, maybe move this section below "Additional elements for handwritten sources" above?

Reply:
I could also imagine joining this section with the section @musicEnfanthen mentioned. The IG created this section because we know this kind of description needs only with older music, but it is not exclusively medical, I agree.

Reply:
I would actually say that these are not exclusive to manuscripts. There are printed books that contain many of these elements (decoration, rubrics, heraldry, etc.). 

I just noticed `<signatures>` here too -- that should be moved to the folia / bifolia section since it describes page folding and layout.

Reply:
- What about renaming this section into "Additional Elements for Describing Sources" and joining this section with the "Additional Material/Information"? If we agree on this, I like to bring it up at the next metadata-meeting (next week).
- I'll add the request for `<signatures>` to https://github.com/music-encoding/metadata-ig/issues/25

Reply:
I would suggest "Additional Elements for Source Description", but yes -- that sounds good.